services/gateway/apis/http-version3/swagger/definitions/transactions.json

```diff
@@ -10,60 +10,162 @@
                 "$ref": "#/definitions/TransactionsStatistics"
             },
             "meta": {
-                "$ref": "#/definitions/Pagination"
+                "$ref": "#/definitions/TransactionStatisticsMeta"
+            }
+        }
+    },
+    "TransactionStatisticsMeta": {
+        "type": "object",
+        "required": [
+            "count",
+            "offset",
+            "total"
+        ],
+        "properties": {
+            "count": {
+                "type": "integer",
+                "example": 10,
+                "minimum": 0,
+                "description": "Number of items returned in the request."
+            },
+            "offset": {
+                "type": "integer",
+                "example": 10,
+                "minimum": 0,
+                "description": "Number of items skipped in the response."
+            },
+            "total": {
+                "type": "integer",
+                "example": 100,
+                "minimum": 0,
+                "description": "Number of items returned in the response."
```

Comment:
Please make this description consistent with the description from other endpoints.